docs/readmes/orc8r/dev_minikube.md

```diff
@@ -139,8 +139,8 @@ A minimal values file is at `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r/examples/minik
 
 This section describes how to install based on local charts. However, you can also install charts from the official chart repositories
 
-- Stable: <https://artifactory.magmacore.org/artifactory/helm/>
-- Test: <https://artifactory.magmacore.org/artifactory/helm-test/>
+- Stable: <https://linuxfoundation.jfrog.io/artifactory/helm-prod/>
+- Test: <https://linuxfoundation.jfrog.io/artifactory/helm-test/>
```

Comment:
Both of these redirect me to a login page. Is it really helm-x or magma-helm-x?

Reply:
missed adapting in the docs commit - thanks for finding this